src/App.css

```diff
@@ -33,10 +33,40 @@
   }
 }
 
-.card {
+ .card {
   padding: 2em;
 }
 
 .read-the-docs {
   color: #888;
 }
+
+
+.cars-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 980px;
+}
+
+.left {
+  flex: 1;
+}
+
+.divider {
+  margin: 0 20px;
+  width: 1px;
+  height: 1000px;
+  background-color: black;
+}
+
+.right {
+  flex: 1;
+}
+.car-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
```

Comment:
This should be also renamed to something that will be more specific to cars list

Reply:
fixed and changed to "..car-form-list"